draft-ietf-cbor-time-tag.md

```diff
@@ -312,7 +322,10 @@ Key -1: Timescale {#key-timescale}
 
 Key -1 is used to indicate a timescale.  The value 0 indicates UTC,
 with the POSIX epoch {{TIME_T}}; the value 1 indicates TAI, with the
-PTP (Precision Time Protocol) epoch {{IEEE1588-2008}}.
+PTP (Precision Time Protocol) epoch {{IEEE1588-2008}}; the value 2
+indicates UT1 (Earth Rotation Angle), as defined by the International
+Earth Rotation and Reference Systems Service (IERS) and illustrated by
+the reference terminology in ANNEX 1 of {{ITU-R_TF.460-6}}.
```

Comment:
This is a bit of a lame reference, as its main content is "Concise definitions of the above terms and the concepts involved are available in the publications of the IERS (Paris, France)."  What are those IERS documents?  We'd need to say this here...